magento/magento2
# Changing products 'this item has weight' using 'Update Attributes' is…

… not possible #6310



### Description
The fixed Is not possible  Changing products 'this item has weight' using 'Update Attributes' 

### Fixed Issues (if relevant)

1. magento/magento2#6310: Issue title Changing products 'this item has weight' using 'Update Attributes' is not possible

### Manual testing scenarios

1. Go to admin panel to Catalog->Products
2. Create "Simple","Virtual" and "Downloadable" products
3. Click any count products checkbox in product grid
4. Check "Update attributes" in action select  from above grid
5. Click radio button "Does this have a weight?"  - "Yes" or  "No"
6. Click Save Button
Expected Result:
If you select  "Yes", all selected products should be type "Simple"
If you select "No", all selected products should be type "Vitrual"

### Contribution checklist
